domains/usamakhalid.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "THIS IS MY PORTFOLIO WEBSITE, But I am still working on it. I will update it soon.",
+    "repo": "https://github.com/usama0155/portfolio",
+    "owner": {
+        "username": "usama0155",
+        "email": "[usamak5453@gmail.com]"
+    },
+    "record": {
+        "CNAME": "https://usama0155.github.io/portfolio/"
```

```suggestion
        "CNAME": "usama0155.github.io"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`).